site/public/js/forum.js

```diff
@@ -1366,6 +1311,47 @@ function toggleLike(post_id, current_user) {
     });
 }
 
+function updateLikesDisplay(post_id, data) {
+    const likes = data['likesCount'];
+    const liked = data['status'];
+    const staffLiked = data['likesFromStaff'];
+
+    const likeCounterElement = document.getElementById(`likeCounter_${post_id}`);
+    let likeCounter = parseInt(likeCounterElement.innerText);
+
+    // eslint-disable-next-line no-useless-concat
+    const likeIconSrc = document.getElementById(`likeIcon_${post_id}`);
+    let likeIconSrcElement = likeIconSrc.src;
+
+    if (liked === 'unlike') {
+        likeIconSrcElement = likeIconSrcElement.replace('on-duck-button.svg', 'light-mode-off-duck.svg');
+
+        if (staffLiked > 0) {
+            $(`#likedByInstructor_${post_id}`).show();
+        }
+        else {
+            $(`#likedByInstructor_${post_id}`).hide();
+        }
+
+        likeCounter = likes;// set to the sql like value
+
+        likeIconSrc.src = likeIconSrcElement; // Update the state
+        likeCounterElement.innerText = likeCounter;
```

Comment:
this too

Reply:
Should be fixed in https://github.com/Submitty/Submitty/pull/9906/commits/a28c2a90a9859e07dfa164cb5bd05dde5c02d42d